include/swift/AST/Attr.def

```diff
@@ -292,6 +292,10 @@ DECL_ATTR(_restatedObjCConformance, RestatedObjCConformance,
 SIMPLE_DECL_ATTR(_downgrade_exhaustivity_check, DowngradeExhaustivityCheck,
                  OnEnumElement | LongAttribute | UserInaccessible, 71)
 
+SIMPLE_DECL_ATTR(_implicitly_unwrapped_optional,
+                 ImplicitlyUnwrappedOptional,
+                 OnFunc | OnVar | RejectByParser, 72)
```

Comment:
Also OnConstructor and OnSubscript, since those can have IUO returns too

Reply:
Yeah I meant to go back and look at this again. Thanks for the reminder. I'm going to try to add some fairly detailed tests once I start adding the code that starts forcing things as appropriate.